### ISSUE TITLE
Angular-cli builds are broken with 4.0.0 (Cannot read property 'VisibilityDirective' of undefined)

### ISSUE THREAD

**Issue by adrm** [swimlane/ngx-datatable#377]
**I'm submitting a bug report.**

**Current behavior**
Builds are broken with last version of ngx-datatable. When trying to build, it errors with this message:

```
Cannot read property 'VisibilityDirective' of undefined, resolving symbol NgxDatatableModule in C:/Users/user/project/fronte
nd/node_modules/@swimlane/ngx-datatable/release/datatable.module.d.ts, resolving symbol NgxDatatableModule in C:/Users/user/pr
oject/frontend/node_modules/@swimlane/ngx-datatable/release/datatable.module.d.ts
Error: Cannot read property 'VisibilityDirective' of undefined, resolving symbol NgxDatatableModule in C:/Users/user/project
/frontend/node_modules/@swimlane/ngx-datatable/release/datatable.module.d.ts, resolving symbol NgxDatatableModule in C:/Users/a
dria/project/frontend/node_modules/@swimlane/ngx-datatable/release/datatable.module.d.ts
    at simplifyInContext (C:\Users\user\project\frontend\node_modules\@angular\compiler-cli\src\static_reflector.js:475:23)
    at StaticReflector.simplify (C:\Users\user\project\frontend\node_modules\@angular\compiler-cli\src\static_reflector.js:4
78:22)
    at StaticReflector.annotations (C:\Users\user\project\frontend\node_modules\@angular\compiler-cli\src\static_reflector.j
s:60:36)
    at AotPlugin.getNgModuleMetadata (C:\Users\user\project\frontend\node_modules\@ngtools\webpack\src\plugin.js:330:41)
    at C:\Users\user\project\frontend\node_modules\@ngtools\webpack\src\plugin.js:345:51
    at Array.map (native)
    at AotPlugin.extractLoadChildren (C:\Users\user\project\frontend\node_modules\@ngtools\webpack\src\plugin.js:343:14)
    at C:\Users\user\project\frontend\node_modules\@ngtools\webpack\src\plugin.js:346:30
    at Array.map (native)
    at AotPlugin.extractLoadChildren (C:\Users\user\project\frontend\node_modules\@ngtools\webpack\src\plugin.js:343:14)
    at AotPlugin._processNgModule (C:\Users\user\project\frontend\node_modules\@ngtools\webpack\src\plugin.js:272:38)
    at C:\Users\user\project\frontend\node_modules\@ngtools\webpack\src\plugin.js:242:39
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Expected behavior**

The build should be successful.

**Reproduction of the problem**

An angular-cli project and last version of ngx-datatable.

@kuncevic also reports this same behaviour [here](https://github.com/swimlane/ngx-datatable/issues/357#issuecomment-267488927).

**Please tell us about your environment:**

```
angular-cli: 1.0.0-beta.22-1
node: 6.3.1
os: win32 x64
```

* **Table version:** 4.0.0

* **Angular version:** 2.2.3

* **Browser:** N/A

* **Language:** TypeScript 2.0.10

**Comment by tennisgent** [comment]
I get this same issue. Please resolve asap. 👍

**Comment by nickraphael** [comment]
I just hit this too.

**Comment by adrm** [comment]
Solved with angular-cli 1.0.0-beta.24 and angular 2.3.1.
